src/scripts/main.js

```diff
@@ -1 +1,66 @@
 'use strict';
+
+let rightButtonClicked = false;
+let leftButtonClicked = false;
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    leftButtonClicked = true;
+    resolve('First promise was resolved');
+  });
```

Comment:
The event listener for the first promise resolves the promise on any click, which means it will resolve immediately on the first click. This may not be the intended behavior if you want to wait for a specific condition before resolving.